app/views/users/show.html.erb

```diff
@@ -1,3 +1,4 @@
 <h1> User <%= @user.full_name %> </h1>
 <h2 class="user-role"> <%= @user.role %> </h2>
+<h2 class="user-intent"> <%= @user.intent %> </h2>
```

Comment:
We may want to make this element contingent on the presence of an intent selection, otherwise it will still render even if there's no visible text.